pandas/tests/io/test_sql.py

```diff
@@ -4358,3 +4358,27 @@ def test_xsqlite_if_exists(sqlite_buildin):
         (5, "E"),
     ]
     drop_table(table_name, sqlite_buildin)
+
+
+@pytest.mark.parametrize("dtype_backend", ["pyarrow", "numpy_nullable", lib.no_default])
+def test_bytes_column(sqlite_buildin, dtype_backend):
```

Comment:
```suggestion
def test_bytes_column(all_connectable, dtype_backend):
```

Should test this against all databases

Reply:
sure thing! I was trying to pass in the cartesian product of the `all_connectable` and `dtype_backend` arrays using `itertools.product` to `@pytest.mark.parametrize` but was running into issues with the connections getting passed. I instead made it so the connectables are being passed in the `parametrize` and then we loop through the dtypes in the test. Would love to know if there's a better way to do this so we're testing each `dtype_backend`/`connection` combination independently